scenarios/aws/eks/fakeintake.go

```diff
@@ -0,0 +1,16 @@
+package eks
+
+import (
+	ddfakeintake "github.com/DataDog/test-infra-definitions/components/datadog/fakeintake"
+	"github.com/DataDog/test-infra-definitions/resources/aws"
+	"github.com/DataDog/test-infra-definitions/resources/aws/ecs"
+)
+
+func NewEcsFakeintake(env aws.Environment) (*ddfakeintake.ConnectionExporter, error) {
```

Comment:
Can you not export this function, ie rename to `NewEcsFakeintake` because e2e user can use it instead of `ecs.NewEcsFakeintake`

Reply:
Good suggestion.
I made this function private: https://github.com/DataDog/test-infra-definitions/commit/ee56648ad2565da9a046af34a93de24d4cd9a09b#diff-a3d0fb2a3a7b9f7cbf5b219226efa06731aadded47235389dd6646941789f4ffR9.